README.md

```diff
@@ -153,50 +153,41 @@ look at a 'standalone' build, which hooks Eclipse OMR up to the its testing syst
 only.
 
 ### Basic configuration and compile
-To build standalone Eclipse OMR, run the following commands from the top of the
-source tree. The top of the Eclipse OMR source tree is the directory that contains
-run_configure.mk.
+To build standalone Eclipse OMR, run the following commands from the root of the
+source tree. For more detailed instructions please read [BuildingWithCMake.md](doc/BuildingWithCMake.md).
 
-    # Generate autotools makefiles with SPEC-specific presets
-    make -f run_configure.mk SPEC=linux_x86-64 OMRGLUE=./example/glue
+    # Create a build directory and cd into it
+    mkdir build
+    cd build
+    
+    #generate the build system using cmake
+    cmake ..
 
     # Build
-    make
+    make -j
```

Comment:
`-j` on its own allows an unbounded level of parallelism, which is probably not what is wanted

Reply:
Fixed